src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -210,6 +210,15 @@ Assembly TryLoadAssembly(AssemblyName assemblyName)
                         return assembly;
                     }
 
+                    // Finds and loads NuGet assemblies if msbuildPath is in a VS installation
```

Comment:
should this be wrapped in an `#if NETFRAMEWORK` or something so we don't waste the time on Core?